diff --git a/player/lua/osc.lua b/player/lua/osc.lua
index 3ba1890..61f6988 100644
--- a/player/lua/osc.lua
+++ b/player/lua/osc.lua
@@ -2654,7 +2654,7 @@ function tick()
             ass:new_event()
             ass:pos(display_w / 2, icon_y + 65)
             ass:an(8)
-            ass:append("Drop files or URLs to play here.")
+            ass:append("Drop files or URLs to play here.\nVVCEasy version\nBuilt by Martin Eesmaa")
         end
         set_osd(display_w, display_h, ass.text, -1000)
 
